packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/TeamPromptResponseSummary.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import {TeamPromptResponseSummary_meeting$key} from 'parabol-client/__generated__/TeamPromptResponseSummary_meeting.graphql'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import getPhaseByTypename from '~/utils/getPhaseByTypename'
+import {isNotNull} from '~/utils/predicates'
+import sortByISO8601Date from '~/utils/sortByISO8601Date'
+import TeamPromptResponseSummaryCard from './TeamPromptResponseSummaryCard'
+
+const ResponseSummaryGrid = styled('div')({
+  display: 'flex',
+  flexWrap: 'wrap',
+  padding: '24px',
+  justifyContent: 'center'
+})
```

Comment:
+1 to make future reuse in emails easier wouldn't it be better to use https://github.com/ParabolInc/parabol/blob/22c7a81694d8d4acb75ea5c57e322d301c828516/packages/client/hooks/useEmailItemGrid.tsx#L10 or is it something you already explored?